src/uct/ib/base/ib_md.h

```diff
@@ -333,7 +334,10 @@ uct_ib_md_is_flush_rkey_valid(uint32_t flush_rkey) {
 
 static UCS_F_ALWAYS_INLINE uint8_t uct_ib_md_get_atomic_mr_id(uct_ib_md_t *md)
 {
-    return md->flush_rkey >> 8;
+    if (md->config.enable_atomic_offset) {
+        return md->flush_rkey >> 8;
+    }
+    return 0;
```

Comment:
seems it breaks remote flush functionality  `UCT_FLUSH_FLAG_REMOTE`  (e.g. see `uct_dc_mlx5_flush_remote_rkey`)

Reply:
in this PR it's not a problem since param is `on` by default, but in future if we make it `off` or `auto` will need to handle this

Reply:
Yes, but this PR introduces new env var, tweaking which may result in a failure.
Maybe just temprorarily disable the tests instead?

Reply:
Shall we use the existing "INDIRECT_ATOMIC" env var?

Reply:
it still breaks remote flush functionality. Can you disabled "REMOTE_FLUSH" capability when this env var is set?